packages/paste-core/components/chat-composer/src/ChatComposer.tsx

```diff
@@ -29,37 +28,27 @@ import {
   HistoryPlugin,
 } from '@twilio-paste/lexical-library';
 import {StylingGlobals} from '@twilio-paste/styling-library';
-import type {LexicalComposerProps, OnChangeFunction, ContentEditableProps} from '@twilio-paste/lexical-library';
+import type {
+  LexicalComposerProps,
+  ContentEditableProps,
+  EditorState,
+  LexicalEditor,
+} from '@twilio-paste/lexical-library';
 import merge from 'deepmerge';
 
 import {chatComposerLexicalStyles} from './styles';
 import {AutoLinkPlugin} from './AutoLinkPlugin';
 import {PlaceholderWrapper} from './PlaceholderWrapper';
 import {baseConfig, renderInitialText} from './helpers';
 
-export interface ChatComposerProps extends ContentEditableProps {
+export interface ChatComposerProps extends Omit<ContentEditableProps, 'style' | 'className' | 'onChange'> {
```

Comment:
Rather than set them to never, just don't allow them. And new onChange types from ContentEditable clash so remove that too